internal/generator/generator.go

```diff
@@ -77,17 +77,10 @@ type statement interface {
 	Statement() (*jen.Statement, error)
 }
 
-type fileMeta struct {
-	fileConfig *FileConfig
-	methods    []*method.Method
-}
-
 // Generated contains the code generation out for all the processed types
 type Generated struct {
 	// Common is the golang code that is shared across all generated types
 	Common string
-	// Constants is the golang code that holds constants for compile-time safe references to the proxied methods
```

Comment:
Can you provide more context on the PR? What's the scenario you're fixing? What do you mean by proxy?

Reply:
This line is being deleted, Christian referred to the reinforced codegen as a proxy to the original iface/struct/etc.